tox.ini

```diff
@@ -2,10 +2,11 @@
 envlist=py2,py3
 
 [pytest]
-addopts=-m 'not functional'
+addopts=-m 'normal'
 markers=
+    normal: mark a test as a normal test.
     functional: mark a test as a functional test.
-    optional: mark a test as an optional test.
+    functional_long_running: mark a test as an long running test in functional.
```

Comment:
This sounds weird. I'd use `mark a test as a long running functional test`.

Reply:
OK, I will change for that.